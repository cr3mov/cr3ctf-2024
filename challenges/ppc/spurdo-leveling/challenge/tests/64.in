27703834456987910 937591123094013262
84 1561800595528878
59 647999
