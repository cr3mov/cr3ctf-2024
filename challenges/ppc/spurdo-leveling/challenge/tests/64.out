48600 583 1309
